hilti/src/ast/node_ref.cc

```diff
@@ -11,7 +11,7 @@ NodeRef::NodeRef(Node& n) : _control(n._control()) {}
 
 Node* NodeRef::_node() const {
     if ( ! _control )
-        throw node_ref::Invalid("accesd to uninitalized node reference");
+        throw node_ref::Invalid("access to uninitialized node reference");
```

Comment:
Not added here, but we should eventually make `node_ref::Invalid` a proper `RuntimeError`.

Reply:
It's a purely compiler/AST-side error, so not really a runtime error I would say.